hbase-server/src/main/java/org/apache/hadoop/hbase/wal/BoundedWriterCreationOutputSink.java

```diff
@@ -40,13 +43,13 @@
  * Bounded means the output streams will be no more than the size of threadpool
  */
 @InterfaceAudience.Private
-public class BoundedLogWriterCreationOutputSink extends LogRecoveredEditsOutputSink {
+public class BoundedWriterCreationOutputSink extends RecoveredEditsOutputSink {
```

Comment:
In general, I prefer we have a base class and then two sub classes, one is for writting out recovered wals, and the other is for writting out recovered hfiles. Making HFile extends WAL is a bit strange.

Reply:
Let me try a better refactor.